src/main/java/com/rocketden/main/exception/ProblemError.java

```diff
@@ -9,7 +9,8 @@
 public enum ProblemError implements ApiError {
 
     BAD_SETTING(HttpStatus.BAD_REQUEST, "Please choose either Easy, Medium, or Hard " +
-            "(or Random if choosing a room difficulty)."),
+            "(or Random if choosing a room difficulty), as well as a number of problems."),
```

Comment:
What are you looking to distinguish -- between the difficulty levels and the number of problems? Should I change this to `NULL_DIFFICULTY` and `NULL_NUM_PROBLEMS` errors?

Reply:
Yeah, in that case that might make more sense (although I would think something like "bad" instead of "null") - that way it can distinguish between bad difficulty setting vs. bad choice for num problems.

Reply:
Ok, I'll make this change.

Reply:
Done! 👍